doc/release-notes-25574.md

```diff
@@ -0,0 +1,12 @@
+Updated settings
+----------------
+
+If the `-checkblocks` or `-checklevel` options are explicitly provided by the
+user, but the verification checks cannot be completed due to an insufficient
+dbcache, bitcoind will now return an error at startup. (#25574)
```

Comment:
nit in 42b192f0cbf9c04da111145c921344b0881b3ce3: 
```suggestion
dbcache, Bitcoin Core will now return an error at startup. (#25574)
```

(holds for bitcoin-qt as well)

Reply:
done